servers/republik/modules/crowdfundings/graphql/resolvers/_mutations/updateUser.js

```diff
@@ -1,18 +1,39 @@
 const { Roles } = require('@orbiting/backend-modules-auth')
 const logger = console
-const updateUser = require('../../../lib/updateUser')
 
 module.exports = async (_, args, {pgdb, req, t}) => {
   Roles.ensureUserHasRole(req.user, 'supporter')
+
   const user = await pgdb.public.users.findOne({id: args.userId})
   if (!user) {
     logger.error('user not found', { req: req._log() })
     throw new Error(t('api/users/404'))
   }
-  return updateUser(args, {
-    user,
-    pgdb,
-    req,
-    t
-  })
+
+  const {firstName, lastName, birthday, address, phoneNumber} = args
+  const transaction = await pgdb.transactionBegin()
+  try {
+    if (firstName || lastName || birthday || phoneNumber) {
+      await transaction.public.users.update({id: user.id}, {
+        firstName,
+        lastName,
+        birthday,
+        phoneNumber
+      }, {skipUndefined: true})
+    }
```

Comment:
Question: If I passed `lastName: undefined` but `firstName: 'Luki'`, wouldn't `lastName` be set to undefined? Not really an issue atm, as the client still expects a complete particulars/address set.